target_postgres/postgres.py

```diff
@@ -407,93 +306,84 @@ def get_update_sql(self, target_table_name, temp_table_name, key_properties, sub
                         insert_distinct_on=insert_distinct_on,
                         insert_distinct_order_by=insert_distinct_order_by)
 
-    def persist_rows(self,
-                     cur,
-                     target_table_name,
-                     temp_table_name,
-                     remote_schema,
-                     streamed_json_schema,
-                     key_properties,
-                     records):
-        headers = list(remote_schema['schema']['properties'].keys())
-
-        datetime_fields = [k for k,v in streamed_json_schema['schema']['properties'].items()
-                           if v.get('format') == 'date-time']
-
-        default_fields = {k: v.get('default') for k, v in streamed_json_schema['schema']['properties'].items()
-                          if v.get('default') is not None}
-
-        fields = set(headers +
-                     [v['from'] for k, v in remote_schema.get('mappings', {}).items()])
+    def parse_table_record_serialize_field_name(self, remote_schema, streamed_schema, field, value):
```

Comment:
@AlexanderMann You're using the word "parse" a lot. Whenever I see that in code I take it as meaning interpreting / deserializing something from a string. I realize the that's not the only use of the word parse, and context can help, but I think it's losing exact meaning in here.